Clarified language about what external drivers are in the context of … (#22633)
qmk/qmk_firmware: docs/feature_rgb_matrix.md

# RGB Matrix Lighting :id=rgb-matrix-lighting

> This feature allows you to use RGB LED matrices driven by external drivers. *External* here means *not within the microcontroller,* and includes LEDs with embedded drivers. RGB Matrix Lighting is commonly used for per-key backlighting. It hooks into the RGBLIGHT system so you can use the same keycodes as RGBLIGHT to control it. 

Reviewer: ```suggestion
This feature allows you to use RGB LED matrices controlled by external LED driver ICs.
```
